src/app/components/repo-list-component/repo-list.component.ts

```diff
@@ -35,8 +35,8 @@ export class RepoListComponent {
         this.navigationService.navigateToOwnerView(ownerName);
     }
 
-    public onSelectDataset(ownerName: string, id: string): void {
-        this.onSelectDatasetEmit.emit({ ownerName, id });
+    public onSelectDataset(row: Dataset): void {
+        this.onSelectDatasetEmit.emit({ ownerName: row.name, id: row.id });
```

Comment:
`row.name` is a name of the dataset, not the owner name.
owner name is `row.owner.name`

Reply:
Fixed